src/Xrefcheck/Verify.hs

```diff
@@ -138,61 +142,90 @@ data VerifyError
 instance Given ColorMode => Buildable VerifyError where
   build = \case
     LocalFileDoesNotExist file ->
-      "⛀  File does not exist:\n   " +| file |+ "\n"
+      [int||
+      ⛀  File does not exist:
+         #{file}
+      |]
 
     LocalFileOutsideRepo file ->
-      "⛀  Link targets a local file outside repository:\n   " +| file |+ "\n"
+      [int||
+      ⛀  Link targets a local file outside repository:
+         #{file}
+      |]
 
     AnchorDoesNotExist anchor similar ->
-      "⛀  Anchor '" +| anchor |+ "' is not present" +|
-      anchorHints similar
+      [int||⛀  Anchor '#{anchor}' is not present|] +| anchorHints similar
 
     AmbiguousAnchorRef file anchor fileAnchors ->
-      "⛀  Ambiguous reference to anchor '" +| anchor |+ "'\n   " +|
-      "In file " +| file |+ "\n   " +|
-      "Similar anchors are:\n" +|
-          blockListF' "    -" build fileAnchors |+ "" +|
-      "   Use of such anchors is discouraged because referenced object\n\
-      \   can change silently whereas the document containing it evolves.\n"
+      [int||
+      ⛀  Ambiguous reference to anchor '#{anchor}'
+         In file #{file}
+         Similar anchors are:
+      #{blockListF' "    -" build fileAnchors}
+          Use of such anchors is discouraged because referenced object
+          can change silently whereas the document containing it evolves.
+      |]
```

Comment:
Let's take the opportunity to improve this message:

```suggestion
         It could refer to either:
      #{indentF 4 $ blockListF' "-" build fileAnchors}
         Use of ambiguous anchors is discouraged because the target
         can change silently while the document containing it evolves.
      |]
```

Reply:
Done